tree/ntuple/v7/test/rfield_vector.cxx

```diff
@@ -302,25 +302,25 @@ TEST(RNTuple, ComplexVector)
       EXPECT_EQ(0, ComplexStruct::GetNCallDestructor());
       EXPECT_EQ(10u, rdV->size());
       ntuple->LoadEntry(2);
-      EXPECT_EQ(10000, ComplexStruct::GetNCallConstructor());
-      EXPECT_EQ(0, ComplexStruct::GetNCallDestructor());
+      EXPECT_EQ(10010, ComplexStruct::GetNCallConstructor());
```

Comment:
Weird... why do we see more calls to the ctor/dtor overall?

Reply:
The current version of vector reading is buggy and forgets necessary calls to the destructor/constructor.